mycroft/skills/twitter/__init__.py

```diff
@@ -0,0 +1,49 @@
+from os.path import dirname
+
+import tweepy
+
+from adapt.intent import IntentBuilder
+from mycroft.skills.core import MycroftSkill
+from mycroft.util.log import getLogger
+
+__author__ = 'crios'
+
+LOGGER = getLogger(__name__)
+
+
+class TwitterSkill(MycroftSkill):
+    def __init__(self):
+        super(TwitterSkill, self).__init__(name="TwitterSkill")
+        ACCESS_TOKEN = self.config.get('aToken')
+        ACCESS_TOKEN_SECRET = self.config.get('aTokenSecret')
+        CONSUMER_KEY = self.config.get('cKey')
+        CONSUMER_SECRET = self.config.get('cSecret')
+        auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+        auth.set_access_token(ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
+        self.api = tweepy.API(auth)
+
+    def initialize(self):
+        self.load_data_files(dirname(__file__))
+        self.register_regex("that (?P<PostText>.*)")
+        self.register_regex("i am (?P<PostText>.*)")
+        twitter_intent = (IntentBuilder("TwitterIntent")
+                          .require("twitterKeyword")
+                          .require("PostText").build())
+        self.register_intent(twitter_intent, self.handle_twitter_intent)
+
+    def handle_twitter_intent(self, message):
+        try:
+            tweet = message.metadata.get("PostText", message)
+            post_text = ((tweet) + ' ' + '#mycroftaiskill')
```

Comment:
Loved the idea of inserting the hashtag :+1: 
What about making this concatenation simpler? Like:
```
tweet = message.metadata.get("PostText", message)

if tweet:
    tweet = '#mycroftaiskill'
    self.api.update_status(status=post_text)
```